### PR TITLE
Update Set-CsExternalAccessPolicy.md

### PR DESCRIPTION
**Pull request by luis-msftpm**
Adding information regarding -EnableTeamsConsumerInbound and -EnableTeamsConsumerAccess. 
This new properties were introduced as part of the effort to enable communications between people using Teams with an account managed by an organization and people using Teams with an account that's not managed by an organization.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Set-CsExternalAccessPolicy.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ms.reviewer: rogupta
...
+External access policies determine whether or not your users can: 1) communicate with users who have Session Initiation Protocol (SIP) accounts with a federated organization; 2) communicate with users who are using custom applications built with [Azure Communication Services](/azure/communication-services/concepts/teams-interop); 3) access Skype for Business Server over the Internet, without having to log on to your internal network; 4) communicate with users who have SIP accounts with a public instant messaging (IM) provider such as Skype; and, 5) communicate with people who are using Teams with an account that's not managed by an organization
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsExternalAccessPolicy.md:16]:
```suggestion
External access policies determine whether or not your users can: 1) communicate with users who have Session Initiation Protocol (SIP) accounts with a federated organization; 2) communicate with users who are using custom applications built with [Azure Communication Services](/azure/communication-services/concepts/teams-interop); 3) access Skype for Business Server over the Internet, without having to log on to your internal network; 4) communicate with users who have SIP accounts with a public instant messaging (IM) provider such as Skype; and, 5) communicate with people who are using Teams with an account that's not managed by an organization.
```

**Review by dariomws — APPROVED** [review]

**Comment by get-itips** [comment]
@luis-msftpm 
Are you a Microsoft employee?

Here is a site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=luis-msftpm

if you have a Microsoft alias you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

**Review by get-itips — APPROVED** [review]
Can you make the same additions to `New-CsExternalAccessPolicy` ?

**Review by tseward — APPROVED** [review]

**Comment by luis-msftpm** [comment]
@get-itips I'm the PM owner of this feature. 
Thanks for the pointer. I've just linked my acct. 

> Can you make the same additions to `New-CsExternalAccessPolicy` ?
Pull Request sent for that one. Done.

**Comment by get-itips** [comment]
Thank you @luis-msftpm

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**